plugins/topdesk/icon_topdesk/util/helpers.py

```diff
@@ -25,9 +25,14 @@ def clean(item_to_clean: Union[dict, list]) -> Union[dict, list]:
 
 
 def prepare_incident_payload(parameters: dict) -> dict:
+    is_status = False
+
     payload = {}
-    for key in parameters.keys():
-        value = parameters.get(key)
+    for key, value in parameters.items():
+        if key == "status":
+            if value != "":
```

Comment:
This could be combined
```suggestion
        if key == "status" and value:
```